src/state/apis/nft/nftApi.ts

```diff
@@ -248,7 +271,8 @@ export const nftApi = createApi({
         dispatch(nft.actions.upsertNfts({ byId: nftsById, ids: Object.keys(nftsById) }))
 
         const collectionsById = nftsWithCollection.reduce<NftState['collections']['byId']>(
-          (acc, item) => {
+          (acc, _item) => {
+            const item = cloneDeep(_item)
```

Comment:
Actually needed because of `TypeError: Cannot add property isSpam, object is not extensible`, and `Object.assign()` won't cut it since we're adding it to `item.collection` nested property, and `Object.assign()` only does a shallow clone